view/frontend/templates/cart/item/renderer/actions/product-protection-simple-offer.phtml

```diff
@@ -0,0 +1,40 @@
+<?php
+/** @var \Magento\Quote\Model\Quote\Item $_item */
+$item = $block->getItem();
+
+/** @var \Magento\Catalog\Model\Product $product */
+$product = $item->getProduct();
+
+/** @var \Magento\Quote\Model\Quote $quote */
+$quote = $item->getQuote();
+
+$productType = $product->getTypeId();
+
+$categoryName = '';
+
+if ($category = $product->getCategory()) {
+    $categoryName = $category->getName();
+}
+?>
+<div>
+    <div class="product-protection-offer" id="product_protection_offer_<?= $product->getSku() ?>"></div>
+</div>
+<script type="text/x-magento-init">
+    {
+        "#product_protection_offer_<?= $product->getSku() ?>": {
+            "simpleProductProtectionOffer": [
+               {
+                        "extendStoreUuid": "<?= $block
+                            ->getData('viewModel')
+                            ->getExtendStoreUuid() ?>",
+                        "activeEnvironment": "<?= $block
+                            ->getData('viewModel')
+                            ->getActiveEnvironment() ?>",
+                        "selectedProductSku": "<?= $product->getSku() ?>",
```

Comment:
If you add a grouped or configurable product to the cart, is this the configurable product data, or the simple product variation?

Reply:
Right now, we're only showing up offer for simple product.

Reply:
It works for Grouped Product and Configurable product.